### PR TITLE
expansion and clarifications

### PR DESCRIPTION
**Pull request by paul-hammant**
You're not liking the issues I raised, so maybe it's a PR you want.

### CODE REVIEW

**Comment by dguo** [comment]
Hey. I did appreciate the issues that you opened, and thanks for the PR. I've just been busy/focusing on other things for the last month. I've been meaning to come back to this.

I'll take a look later tonight.

**Comment by paul-hammant** [comment]
:)

File: index.html [hunk 1]
```diff
@@ -332,11 +335,22 @@ <h3 class="title is-4">
...
+                <br>
+
+                <h3 class="title is-4">
+                    Contributing back
                 </h3>
                 <p>
                     If you are open to contributions from others, let them know
-                    what your requirements are for accepting contributions.
+                    what your requirements are for accepting contributions. Perhaps
+                    also how keen you are to process pull-requests, and issues.
```
Comment (dguo) [inline index.html]: Isn't this sort of covered already by the last section regarding slowed down development?

File: index.html [hunk 2]
```diff
@@ -358,6 +372,18 @@ <h3 class="title is-4">
...
+                <h3 class="title is-4">
+                    Slowdown in project activity or ceasing of development activity
+                </h3>
+                <p>
+                    If you have run out of energy for the project, but other people might
+                    still find it some how and get excited initially, put a note at the
+                    top of the README, that development has slowed down, or stopped completely
+                    if that is the case.  GitHub allows forks, and you may be pleased if one
+                    of those takes off.
+                    <a href="#license-1">licensed</a>.
```
Comment (dguo) [inline index.html]: It looks like this line was left over from a copy/paste. It can be removed.

File: index.html [hunk 3]
```diff
@@ -358,6 +372,18 @@ <h3 class="title is-4">
                     For open source projects, say how it is
                     <a href="#license-1">licensed</a>.
                 </p>
+
+                <h3 class="title is-4">
+                    Slowdown in project activity or ceasing of development activity
+                </h3>
+                <p>
+                    If you have run out of energy for the project, but other people might
```
Comment (dguo) [inline index.html]: What do you think of:

If you have run out of time or energy for your project, put a note at the top of the README that development has slowed down or stopped completely. Someone may choose to fork your project or volunteer to step in as a maintainer or owner, allowing your project to keep going.

**Comment by paul-hammant** [comment]
I'd have just consumed the PR and followed it with a commit myself if I had preferences. The risk of otherwise good contribs being left hanging is too high otherwise ;)

**Comment by dguo** [comment]
I considered doing that but wanted to give you a chance to voice your opinion. If you would prefer I just merge and make changes, I'll do that.

**Comment by paul-hammant** [comment]
Thank you - I wrote about a  3-time former colleague @joewalnes here: https://paulhammant.com/2014/05/26/joe-walnes-on-successful-open-source/ (and his influential open source pieces). 

Section #3 "getting others involved", I quote :-
 
.8: Implore potential collaborators to contribute patches (pull requests) that are implementation and tests
.9: Rewrite/Refactor as you speedily consume patches

:-)

**Comment by paul-hammant** [comment]
Do you have a netlify daemon watching the development branch and deploying to something like https://development.makeareadme.com/ ?

Netlify (and their LetsEncrypt fu) is pretty cool, huh?

**Comment by dguo** [comment]
I didn't, but now I do! Good idea. And yep, Netlify is awesome.

I made some edits and pushed to master. It's live now.

**Pull request merged**